sdks/go.mod

```diff
@@ -23,33 +23,34 @@ module github.com/apache/beam/sdks/v2
 go 1.16
 
 require (
-	cloud.google.com/go/bigquery v1.17.0
-	cloud.google.com/go/datastore v1.5.0
-	cloud.google.com/go/pubsub v1.11.0-beta.schemas
-	cloud.google.com/go/storage v1.15.0
+	cloud.google.com/go/bigquery v1.28.0
+	cloud.google.com/go/compute v1.5.0 // indirect
+	cloud.google.com/go/datastore v1.6.0
+	cloud.google.com/go/iam v0.2.0 // indirect
+	cloud.google.com/go/pubsub v1.18.0
+	cloud.google.com/go/storage v1.21.0
```

Comment:
OK so these are being updated. Did you call `go get` explicitly?

`go mod tidy` shouldn't be updating things AFACIT from the docs, unless we are actually changing/adding deps. https://go.dev/doc/modules/managing-dependencies

But I might be mis-reading it.

Reply:
Yeah I had called `go get`.